### ISSUE TITLE
`project delete` does not show project when there is only one project

### ISSUE THREAD

**Issue by thingforward** [thingforward/slyft-cli#9]
```
$ bin/slyft p delete
Are you sure? [no]: no
ERRO 001 Good decision!
```

At least the project name should be shown. I think this is due to chooseProject, which automatically chooses the one that is there if there's only this one project.

**Comment by sillyfellow** [comment]
if only one project, confirm with `"are you sure to delete 'project-name'"`

**Comment by sillyfellow** [comment]
fixes the issue with 582d6fc8f1694eed71ae0646351a0e3f41389f13
